apis/events/protobuf/v1/event.proto

```diff
@@ -55,6 +57,10 @@ enum EventAction {
   Shutdown = 18;
   Start = 19;
   Stop = 20;
+  SubsystemPause = 21;
```

Comment:
How is the initial pause event sent?

Reply:
I didn't add pause when we receive the pause request. I am creating a pause event when its completed. If there is error I am adding error to the event details. Else I am just adding duration for the pause action. So if there is not error it is assumed to be successful.

Reply:
Or should I add an event when pause request is received? And another event when the pause fails?

Reply:
Yes, 1 when it's started, and 1 when it's finished.
Reason for this is that pause sometimes gets stuck, in which case we'd never get the event, so would be good to know if a pause was started at some point or not.

Reply:
Added the events when subsystem pause started and completed.